src/formulas/tokenizer.ts

```diff
@@ -227,21 +224,23 @@ function tokenizeInvalidRange(chars: TokenizingChars): Token | null {
 class TokenizingChars {
   private text: string;
   private currentIndex: number = 0;
+  current: string;
```

Comment:
might wanna have `private _current` and `get current() { return this._current; }` to avoid messing everything up by mistake. If it doesnt impact the perfs

Reply:
I must admit I thought about, and didn't try. But it will inevitably impact the perf.
And since it's a very specialized data structure used only once here. I think it's ok to be less safe for the performance's sake